22280.0.txt

```diff
@@ -0,0 +1,69 @@
+@0
+
+
+@1
+MOV UP, DOWN
+
+# -->
+# CHECKS WHAT S IS
+
+# PASSES NUMBER OF
+# LINES THAT NEED
+# TO BE JUMPED IN
+# THE NEXT BOX TO
+# EXECUTE CORRECT
+# COMMANDS
+
+@2
+LOOP:
+MOV UP, ACC
+JEZ ZERO
+JGZ POS
+JLZ NEG
+#PASS NUM OF LINE
+POS:
+MOV 1, DOWN
+JMP LOOP
+NEG:
+MOV 4, DOWN
+JMP LOOP
+ZERO:
+MOV 7, DOWN
+JMP LOOP
+
+@3
+MOV UP, DOWN
+
+@4
+
+
+@5
+MOV UP, RIGHT
+
+@6
+LOOP:
+JRO UP
```

Comment:
I think this use of `JRO` is overkill for this puzzle -- understanding it will be useful if you keep playing though!